packages/api/README.md

```diff
@@ -1,3 +1,27 @@
 # Meticulous API
 
-This packages contains type definitions used by the Meticulous API.
+This packages contains type definitions that are persisted in the database or S3 and also used in the SDK.
```

Comment:
:thinking: I think reading this README as an external user may be confusing.

Reply:
Yeah, I think that's OK though. We'll want to carefully handhold any changes by external users to this package, since changes here change what's stored in our DBs.